### PR TITLE
First version of API - Feedback needed!

### PR DESCRIPTION
**Pull request by emaraschio**
Hello Alex, 

Here is the first pull request, just I did the rails setup and I created some models and the specs.

You can do: `bundle exec rspec` to run it

### CODE REVIEW

**Comment by emaraschio** [comment]
@alex-nexus I added a few more iterations here, please when you have a chance take a quick look. thanks!

File: db/migrate/20170606211022_create_albums.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CreateAlbums < ActiveRecord::Migration[5.1]
+  def change
+    create_table :albums do |t|
+      t.string :name
+      t.string :art
+      t.string :abstract
+      t.references :artist, foreign_key: true
+      t.references :user, foreign_key: true
```
Comment (alex-nexus) [inline db/migrate/20170606211022_create_albums.rb]: is this `user` necessary?

File: db/migrate/20170606211211_create_songs.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class CreateSongs < ActiveRecord::Migration[5.1]
+  def change
+    create_table :songs do |t|
+      t.string :name
+      t.time :duration
+      t.references :album, foreign_key: true
+      t.references :artist, foreign_key: true
+      t.references :user, foreign_key: true
```
Comment (alex-nexus) [inline db/migrate/20170606211211_create_songs.rb]: again, what's user's usage?

File: spec/factories/albums.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :album do
+    name { Faker::Lorem.word }
+    art { Faker::Lorem.sentence }
+    abstract { Faker::Hipster.sentence }
+    artist_id nil
```
Comment (alex-nexus) [inline spec/factories/albums.rb]: would you consider using object instead of id?

**Comment by alex-nexus** [comment]
A couple things: 
1. I asked some trivial questoins but for v1, let's merge the first pr
2. I will try to ask some questions via issues.

File: db/migrate/20170606211022_create_albums.rb (hunk 1 above)
Comment (emaraschio) [inline db/migrate/20170606211022_create_albums.rb]: My mistake, the user will be the owner only of the playlists and that's all.

File: spec/factories/albums.rb (hunk 3 above)
Comment (emaraschio) [inline spec/factories/albums.rb]: you are right, I will change it.

**Comment by emaraschio** [comment]
Ok cool @alex-nexus

**Pull request merged**